laravel/app/Console/Commands/PushSchedule.php

```diff
@@ -173,13 +172,43 @@ public function handle()
         //Get the array that contains all assigments for day + 1.
 		$all_assns = self::updateAssignments($date);
 
+
         //print all assignments to path
         foreach($all_assns as $assignemnts) {
             fputcsv($fp, $assignemnts);
         }
 
         fclose($fp);
 
+        $column_name = "";
+
+        if (count($all_assns) > 0){
+            $cur_assignment = $all_assns[0];
+            $count = (count($cur_assignment));
+        } else {
+            $count = 0;
+        }
+
+        while($count > 0){
+	        $rem = $count % 26;
+    
+            if ($rem == 0) {
+    	        $column_name .= "Z";
+                $count = ($count/26) - 1;
+            } else {
+    	        $column_name .= chr($rem + 833);
+                $count = $count/26;
+                if ($count < 1) {
+                    $count = 0;
+                }
+        	}
+        }
+
+        $column_name = strrev($column_name);
+        $row_number = count($all_assns);
```

Comment:
Not sure if this should be count($all_assns) +1 because the first row in the file is headers for the columns - line 169.